spec/market_spec.rb

```diff
@@ -0,0 +1,114 @@
+require 'spec_helper'
+
+RSpec.describe Market do
+  before(:each) do
+    @peach = Item.new({name: 'Peach', price: "$0.75"})
+    @tomato = Item.new({name: 'Tomato', price: '$0.50'})
+    @egg = Item.new({name: 'Egg', price: "$0.80"})
+    @bacon = Item.new({name: 'Bacon', price: '$1.50'})
+
+    @vendor1 = Vendor.new("Rocky Mountain Fresh")
+    @vendor2 = Vendor.new("Bob's Fish Market")
+
+    @vendor1.stock(@peach, 25)
```

Comment:
Stocking items might not be a part of your setup, but it works here! Just something to think about - I'd have liked to see the stock method tested in isolation.